src/WebJobs.Script.Grpc/MessageExtensions/GrpcMessageConversionExtensions.cs

```diff
@@ -36,11 +38,18 @@ public static object ToObject(this TypedData typedData)
                 case RpcDataType.Json:
                     return JsonConvert.DeserializeObject(typedData.Json, _datetimeSerializerSettings);
                 case RpcDataType.Http:
-                    return GrpcMessageExtensionUtilities.ConvertFromHttpMessageToExpando(typedData.Http);
+                    return GrpcMessageExtensionUtilities.ConvertFromHttpMessageToExpando(typedData.Http, sharedMemoryManager);
                 case RpcDataType.Int:
                     return typedData.Int;
                 case RpcDataType.Double:
                     return typedData.Double;
+                case RpcDataType.SharedMemoryData:
+                    var sharedMemData = typedData.SharedMemoryData;
+                    string mmfName = sharedMemData.MemoryMappedFileName;
+                    long offset = sharedMemData.Offset;
+                    long count = sharedMemData.Count;
+                    string type = sharedMemData.Type;
+                    return sharedMemoryManager.TryGetAsync(mmfName, offset, count).GetAwaiter().GetResult();
```

Comment:
Moving this under ParameterBinding - will avoid making this a blocking call as well.